test/util/SignatureLibTest.js

```diff
@@ -0,0 +1,176 @@
+// JS Libraries
```

Comment:
I would advice having 1 test file per function. 
e.g.: SignatureLibHashInterestRequestTest.js, SignatureLibHashInterestResponseTest.js, etc
This will keep our test files smaller, easier to handle and modify.

Reply:
Understood, tests have been split now